docs/podman-search.1.md

```diff
@@ -73,8 +73,8 @@ Specific registry to search (only the given registry will be searched, not the d
 **--tls-verify**
 
 Require HTTPS and verify certificates when contacting registries (default: true). If explicitly set to true,
-then tls verification will be used, If set to false then tls verification will not be used. If not specified
-both insecured and default registries will be searched through, and tls will be used when possible.
+then tls verification will be used, If set to false then tls verification will not be used if needed. If not specified
+default registries will be searched through, and tls will be skipped if a registry is listed in the insecure registries.
```

Comment:
Could you add the default registries.conf  /etc/containers/registries.conf

Reply:
Is this how Docker handles tls-verify?  I think we need to be in step with them.